tree/ntuple/v7/inc/ROOT/RPageStorageDaos.hxx

```diff
@@ -28,6 +28,40 @@
 #include <memory>
 #include <string>
 
+using ntuple_index_t = std::uint32_t;
+namespace ROOT::Experimental::Detail {
+struct RDaosNTupleAnchor;
+}
+
+struct RDaosContainerNTupleLocator {
+   std::string fName{};
+   std::uint32_t fHashedName{};
+   ntuple_index_t fIndex{};
+   std::unique_ptr<ROOT::Experimental::Detail::RDaosNTupleAnchor> fAnchor;
```

Comment:
I see that this is either initialized in one of the constructors, or remains a `nullptr`.  Perhaps it can be replaced by `std::optional<ROOT::Experimental::Detail::RDaosNTupleAnchor>`?